openapi.yaml

```
@@ -5560,6 +5560,140 @@ paths:
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: 'All applications by service UUID.'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  properties: { uuid: { type: string, example: app-uuid-123 }, name: { type: string, example: n8n }, human_name: { type: string, example: 'N8N Workflow Automation' }, description: { type: string, example: 'Production workflow automation server' }, fqdn: { type: string, example: 'n8n.example.com:5678' }, image: { type: string, example: 'docker.n8n.io/n8nio/n8n:latest' }, status: { type: string, example: running }, exclude_from_status: { type: boolean, example: false }, is_log_drain_enabled: { type: boolean, example: false }, is_gzip_enabled: { type: boolean, example: true }, is_stripprefix_enabled: { type: boolean, example: true } }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)